src/main/java/com/stumeet/server/common/util/model/ValidImageContentType.java

```diff
@@ -0,0 +1,27 @@
+package com.stumeet.server.common.util.model;
+
+public enum ValidImageContentType {
+	JPG("image/jpg"),
+	JPEG("image/jpeg"),
+	PNG("image/png")
+	;
+
+	private final String contentType;
+
+	ValidImageContentType(String contentType) {
+		this.contentType = contentType;
+	}
+
+	private String getContentType() {
+		return contentType;
+	}
+
+	public static boolean hasContentType(String contentType) {
+		for (ValidImageContentType type : ValidImageContentType.values()) {
+			if(type.getContentType().equalsIgnoreCase(contentType)) {
+				return true;
+			}
+		}
+		return false;
```

Comment:
P2 : 다음과 같이 Stream 구문으로 변경할 수 있을 것 같습니다 👍 

```suggestion
return Arrays.stream(ValidImageContentType.values())
             .anyMatch(type -> type.getContentType().equalsIgnoreCase(contentType));
```